install/froxlor.sql

```diff
@@ -703,7 +703,7 @@ opcache.interned_strings_buffer'),
 	('panel', 'customer_hide_options', ''),
 	('panel', 'is_configured', '0'),
 	('panel', 'version', '0.10.10'),
-	('panel', 'db_version', '201912310');
+	('panel', 'db_version', '201912312');
```

Comment:
You said this RP depends on #783 but you are not using the source code of it, hence the conflicts now

Reply:
Yeah, I'm honestly not sure how you/in general handle multiple features with DB changes at once without conflicting with each other. I've each time used the `master` branch for each PR (without my other changes) - this way it would have been easier to merge PRs with less changes on each own, without depending on each other code-wise.

I've solved the conflicts.

Reply:
Yes basically that's correct of course. But if you depend on other features you would use the the other features code-base as "master" - anyway, these "conflicts" are not unfixable but it's of course nicer if there is less code-trouble :)